packages/next/server/web-server.ts

```diff
@@ -397,6 +398,10 @@ export default class NextWebServer extends BaseServer<WebServerOptions> {
   ): Promise<void> {
     res.setHeader('X-Edge-Runtime', '1')
 
+    if (process.env.__NEXT_BOOT_DEBUG) {
+      res.setHeader('X-Next-Boot', isServerCold() ? 'cold' : 'warm')
```

Comment:
Do we need to mange this per-route? mark it as warm for specific page

Reply:
Hmm I guess it depends on how much time is spent when rendering on initialising the server vs initialising a page